web/src/message_list_view.ts

```diff
@@ -427,7 +427,7 @@ function maybe_restore_focus_to_message_edit_form(): void {
     }, 0);
 }
 
-function is_search_view(): boolean {
+export function is_search_view(): boolean {
     const current_filter = narrow_state.filter();
     if (current_filter && !current_filter.supports_collapsing_recipients()) {
         return true;
```

Comment:
This probably isn't a good choice of function interface to export; I feel like it'd be better to have the caller pass in the filter with this name, or maybe better, have a prep commit moving it to `narrow_state.is_search_view`, which would be more obviously make it clear it's a statement about the current view.

(And there, you'd need a check for whether `current_filter` is defined).

Reply:
done